drf_chunked_upload/settings.py

```diff
@@ -32,5 +33,9 @@
 
 # Max amount of data (in bytes) that can be uploaded. `None` means no limit
 DEFAULT_MAX_BYTES = None
-MAX_BYTES = getattr(settings, 'DRF_CHUNKED_UPLOAD_MAX_BYTES', DEFAULT_MAX_BYTES)
+MAX_BYTES = getattr(
+    settings, 'DRF_CHUNKED_UPLOAD_MAX_BYTES', DEFAULT_MAX_BYTES)
 
+# Boolean indiacating whether only authenticated users should be able to use the upload functionality.
+RESTRICT_TO_AUTH_USERS = getattr(
+    settings, 'DRF_CHUNKED_UPLOAD_RESTRICT_TO_AUTH_USERS', DEFAULT_MAX_BYTES)
```

Comment:
Ignore the previous review; it wasn't properly linked to the commit line. Sorry about that!

Shouldn't this have a different default value than DEFAULT_MAX_BYTES?

Reply:
Ya sorry about that, let me open a new pull request.